### DISCUSSION TITLE
初始化音源时报错：Cannot read properties of null (reading 'statusCode')

### DISCUSSION THREAD

**Discussion by syrf109475** [lyswhut/lx-music-desktop#1874] [Q&A]
初始化"六音音源"时报错：Cannot read properties of null (reading 'statusCode')
经过测试，该音源文件在手机上和虚拟机系统中使用2.7.0客户端初始化均正常，但在实体机系统中报错。
实体机系统：Windows 11 23H2 (build 10.0.22631.3527)
我无法确认是系统问题还是软件问题还是音源文件问题。在此请教大佬们提供解决方案

**Comment by yingfan423** [comment]
作者都说了，2.7.0版本会出现音源初始化失败的现象，可以退回到2.5.0版本，或者寻找新的音源，b站有

**Comment by ttsytang** [comment]
作为软件版本迭代，应该向下兼容啊，不能说升级版本后，以前的音乐源不能用了，这不是扯淡嘛

**Reply by ttsytang** [reply]
目前稳定使用的是 v2.5.0+sixyin-source-v1.0.7

**Comment by lyswhut** [comment]
出于某些原因，故意移除兼容层的：https://github.com/lyswhut/lx-music-desktop/issues/1762#issuecomment-1920456798
